tests/microbenchmarks/bench_moe/routing/materialize.py

```
@@ -102,6 +102,138 @@ def _pack_slots_column_major(flat: List[int], local_num_tokens: int, top_k: int)
+                chosen.append(g)
+        if len(chosen) < topk_group:
+            for g in sorted(range(n_group), key=lambda gg: (-sum(pools[gg].values()), gg)):
+                if g not in chosen:
+                    chosen.append(g)
+                if len(chosen) == topk_group:
+                    break
+        quota = base_quota[t % len(base_quota) :] + base_quota[: t % len(base_quota)]
+
+        row: List[int] = []
+        deficit = 0
+        for g, want in zip(chosen, quota):
```

Ruff (0.16.1)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)